proto/wechaty/puppet/message.proto

```diff
@@ -48,6 +48,10 @@ message MessagePayloadResponse {
   repeated    string mention_ids = 9;
 }
 
+message FileBoxMessage {
```

Comment:
It's great to have a common type for our FileBox related response!

I'd like to suggest that rename the `FileBoxMessage` to `FileBoxChunk`